stan/math/prim/prob/gumbel_lpdf.hpp

```diff
@@ -100,7 +93,8 @@ return_type_t<T_y, T_loc, T_scale> gumbel_lpdf(const T_y& y, const T_loc& mu,
     }
     if (!is_constant_all<T_scale>::value) {
       ops_partials.edge3_.partials_
-          = -y_minus_mu_over_beta * scaled_diff - inv_beta;
+          = -std::move(y_minus_mu_over_beta) * std::move(scaled_diff)
+            - std::move(inv_beta);
```

Comment:
Nor do these.

Reply:
Yeah my b I think I was doing something silly here when testing this out